packages/react-composites/src/composites/localization/LocalizationProvider.tsx

```diff
@@ -69,6 +69,10 @@ export type LocalizationProviderProps = {
  */
 export const LocalizationProvider = (props: LocalizationProviderProps): JSX.Element => {
   const { locale, children } = props;
+  /* beta:start */
+  console.log('beta-test1');
```

Comment:
you will need to remove those, right?

Reply:
You are right! That's why I marked this as POC, this is an example of how things work for normal statements